SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityScheduler.java

```diff
@@ -695,6 +696,10 @@ private int numInstancesExpected(SingularityRequest request, SingularityPendingR
           } else {
             scheduleFrom = new Date(now);
             final CronExpression cronExpression = new CronExpression(request.getQuartzScheduleSafe());
+            Optional<TimeZone> scheduledTimeZone = request.getScheduledTimeZone();
+            if (scheduledTimeZone.isPresent()) {
+              cronExpression.setTimeZone(scheduledTimeZone.get());
+            }
```

Comment:
nitpick, but the variable here is probably unnecessary. also, if the variable is needed, try to make all your variables `final` unless they definitely need to be mutable (see above for an example)